### PR TITLE
Feature/continuous integaration

### PR DESCRIPTION
**Pull request by tuckerbuchy**
I've added some CICD stuff for travis and heroku to deploy this on pushes to master.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,11 +52,12 @@
...
-    "start": "webpack-dev-server",
-    "build": "webpack -d --progress --colors --config ./webpack.config.js",
-    "release": "NODE_ENV=production webpack -p --progress --colors --config ./webpack.config.js"
+    "test": "$(npm bin)/jest --config ./.jestrc",
+    "coverage": "$(npm bin)/jest --config ./.jestrc --coverage",
+    "lint": "$(npm bin)/eslint --ignore-pattern=\"*__tests__*\" src",
+    "start": "$(npm bin)/webpack-dev-server",
+    "build": "$(npm bin)/webpack -d --progress --colors --config ./webpack.config.js",
+    "heroku-postbuild": "webpack -p --config ./webpack.config.js --progress",
+    "release": "NODE_ENV=production $(npm bin)/webpack -p --progress --colors --config ./webpack.config.js"
```
Comment (dylanmccall) [inline package.json]: I'm confused by this bit. Are you sure it's needed? `npm run` already [adds the value of npm bin to the PATH](https://docs.npmjs.com/cli/run-script). It looks like Heroku's own utilities [do the same thing](https://devcenter.heroku.com/articles/nodejs-support#runtime-behavior). If this isn't happening somewhere, I suspect that thing is very broken and should be fixed.

Comment (tuckerbuchy) [inline package.json]: hmm let me try it without, it was failing on me before, but could have been for another reason.

Comment (tuckerbuchy) [inline package.json]: word that works! nice find. Must have been a carry over from when i was experimenting.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+script:
+- npm test
+deploy:
+  provider: heroku
+  api_key:
+    secure: nby0ZMA+xVXEYEisZEBQ8OOec+nh100JqmLErJSsAnSUEC5hqSphM6OKbIitEn61pYZrjCyWTzRwN5p5D2ryX3qOizY64lgBwoeOgF6gFLtqdQCwl4LShjQR6g3tQtXLc4HQFBzVmdSZB8ulrVOiU5Oj9X7b31jBOH+AEsQ0/QUHQ/3fBgxAPs/6bFHFzcTI/fyGrr5xsJKgkD4hmT9hLf/ckNGNSs4JK70x/UBVu2bQR+LMptOMVBPjsfAE6P3W6WqUAKYJoBjxGi3t4My8fcVR4JVKcp+WXSJv3U4M6Z3NEUs2o4VxqzGTlnGR5q8BJZdHrGHRbJ3/xn2BJ5bihpRXH7cdcvYopzHwUxGM8K/mBQylwadImjaAegQQ+oSmz/XvNsuaNDbNcGLmQUBxUxmHzsuVHTAIuJY8xaYtNCs9eMnezWY14apafAkDO2HQvmpOj/c88o8kT9XUjM2kY6Iv2RWdH9HuxScj0zuRfXYW11tOwgTw13yZM/7Dz/XsxcBajkUflRVUh0mXfOugS9rkzbPLAmUghOY9QXZsELLwEnLElSAK5JYnnlt3v7Wg6IKNfHm2Ur8EeblD7AOvX/wHCNGhmxSiY+eUUgsuZtAKWK/gZ7cbIEZkKCXp7QdyGxICylQvj5Ejgvvh5tirgp0yjof9hUzAhjjoqN3Y0sc=
+  app: pathways-frontend
+  on:
+    repo: pg-irc/pathways-frontend
+    all_branches: true
```
Comment (tuckerbuchy) [inline .travis.yml]: gotta remove this, just have it here for testing it

**Pull request merged**